assests/javaScript/movieDetail.js

```diff
@@ -0,0 +1,47 @@
+const movieSection = document.getElementById("movie-detail");
+
+const urlSearchParams = new URLSearchParams(window.location.search);
+const movie_id = urlSearchParams.get("id");
```

Comment:
Always use camel case for variables name.
```suggestion
const movieId = urlSearchParams.get("id");
```

Reply:
yessss thanks